docroot/modules/custom/mass_metatag/mass_metatag.module

```diff
@@ -364,16 +364,17 @@ function mass_metatag_page_attachments(array &$attachments) {
     'resources',
   ];
 
-  $current_uri = Url::fromRoute('<current>')->toString();
```

Comment:
I would think that this returns an aliased URL. Thats not what you are seeing?

Reply:
This is the issue I am trying to resolve. If I am on `http://mass.local/unemployment-insurance-ui-for-employers/tasks` then `Url::fromRoute('<current>')->toString();` returns just `/unemployment-insurance-ui-for-employers`. I was looking for a way to determine if this is a `view_mode_page` and get the "tasks" part but I could't figure out how to that.

Reply:
Oh. In that case 'tasks' is the is called an "argument" or a "param". You can get those from some service. The routematch service I think. I'll look around if you tried that already.

Reply:
Yeah I tried `\Drupal::routeMatch()->getParameter('node')` and got the node entity but I can't find the currently used view_mode in there. The view_mode is `all_tasks` which I assume is exclusively used for the `view_mode_page`

Reply:
Is it in -`>getParamaters()`?

Reply:
I tried that too. The only param is the node.

Reply:
Looking at this again I don't think I need to get the alias. I can probably just use `$current_path = \Drupal::service('path.current')->getPath();` . Even though it returns `/unemployment-insurance-ui-for-employers/tasks` for http://mass.local/unemployment-insurance-ui-for-employers/tasks and `/node/7426/related` for http://mass.local/unemployment-insurance-ui-for-employers/related

Reply:
OK @weitzman I removed getting the alias like I mentioned in the previous comment.